bootstrap

```diff
@@ -84,13 +86,38 @@ extract(Binary) ->
 request(Url) ->
     case httpc:request(get, {Url, []},
                        [{relaxed, true}],
-                       [{body_format, binary}]) of
+                       [{body_format, binary}],
+                       rebar) of
         {ok, {{_Version, 200, _Reason}, _Headers, Body}} ->
             {ok, Body};
         Error ->
             Error
     end.
 
+get_rebar_config() ->
+    {ok, [[Home]]} = init:get_argument(home),
```

Comment:
Yeah I just realized and tried to flush my comments. Saw it was fine later in the actual file.